deployments/k8s/helm/signalfx-agent/values.yaml

```diff
@@ -23,6 +23,14 @@ rollingUpdateMaxUnavailable: 1
 # Namespace to deploy agent in (Optional: Will default to release namespace)
 namespace:
 
+# Specify PSP (Pod Security Policy) (Optional: default to not using PSP mode if no values are provided by the user)
+pspMode:
+
+if $pspMode == 'restricted'
```

Comment:
[Here](https://github.com/helm/charts/blob/master/stable/apm-server/templates/clusterrole.yaml#L1) is an example of conditionalizing the template.  This is yaml, so if statements aren't allowed here.

Reply:
Thanks Ben for giving hint. Sorry for late update. I also come across this Helm Flow control standard (https://github.com/helm/helm/blob/master/docs/chart_template_guide/control_structures.md) and updated value YAML code accordingly with IF condition. Is this sounds good to approve?